.github/workflows/msrv.yml

```diff
@@ -30,16 +30,19 @@ jobs:
       - uses: Swatinem/rust-cache@v2
 
```

Comment:
There's one way to fix this without bumping MSRV:

Before the line rust-cache, do a

```
cargo +nightly update -Zminimal
```

That will guarantee that the version specified in our `Cargo.toml` is used.

It needs to be done before rust-cache, to make sure it picks up the Cargo.lock generated and use it when computing cache key.

Reply:
Thanks for the pointer! Just added this, hopefully it works 🤞